tests/xml/XMLBearTest.py

```diff
@@ -18,14 +20,59 @@
 <a>hey and hi</a>
 """.splitlines(keepends=True)
 
-InvalidXMLCorrectedTest = verify_local_bear(
-                                        XMLBear,
-                                        valid_files=(valid_xml_file,),
-                                        invalid_files=(invalid_xml_file,),
-                                        tempfile_kwargs={"suffix": ".xml"})
-
-InvalidXMLSyntaxTest = verify_local_bear(
-                                        XMLBear,
-                                        valid_files=(valid_xml_chars,),
-                                        invalid_files=(invalid_xml_chars,),
-                                        tempfile_kwargs={"suffix": ".xml"})
+dtd_file = os.path.join(os.path.dirname(__file__),
+                        "test_files",
+                        "note.dtd")
+
+schema_file = os.path.join(os.path.dirname(__file__),
+                           "test_files",
+                           "note.xsd")
+
+valid_xml_path = list(open(os.path.join(os.path.dirname(__file__),
```

Comment:
just to be clear, this already takes up 4 lines. Adding the file directly here is just 5 lines. .. quite ok IMO.

Also ... In case you decide to use this `open` thing and not copy the contents here,
do:

```
valid_xml_url = list(open(os.path.join(
    os.path.dirname(__file__), "test_files", "concept-valid.xml"), 'r'))
```
as its easier to read